src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/PrisonerDetailDto.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.dto
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Prisoner details - only offenderNo and location needed")
+open class PrisonerDetailDto(
+  val offenderNo: String,
```

Comment:
I think we call this prisoner id in VSiP, could use annotation to match it up with prisoner api , ur call